deepchecks/nlp/checks/train_test_validation/text_embeddings_drift.py

```diff
@@ -0,0 +1,145 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2023 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+"""Module contains the embeddings drift check."""
+
+from deepchecks.core import CheckResult, ConditionCategory, ConditionResult
+from deepchecks.nlp import Context, TrainTestCheck
+from deepchecks.nlp.checks.multivariate_embeddings_drift_utils import run_multivariable_drift_for_embeddings
+from deepchecks.utils.strings import format_number
+
+__all__ = ['TextEmbeddingsDrift']
+
+
+class TextEmbeddingsDrift(TrainTestCheck):
+    """
+    Calculate drift between the entire train and test datasets using a model trained to distinguish between them.
```

Comment:
```suggestion
    Calculate drift between the train and test datasets using a model trained to distinguish between them in embedding space.
```
If not more than one line...

Reply:
It is though :( was cutting it very close. 
How about "Calculate drift between the train and test datasets using a model trained to distinguish between their embeddings."?

Reply:
👍